### PR TITLE
upload men fashion from database #37

### PR DESCRIPTION
**Pull request by Salam-Dalloul**
### Fixes
fixes #<issue_number>

### Related
related to #<37>

### Summary
upload men fashion from database
### Major Changes
[**OPTIONAL for small PRs**] Detailed description of major changes in the PR, ideally with references to relevant commits, issues, line-numbers, etc.

### CODE REVIEW

**Comment by Ghadeer93** [comment]
Just one note :relaxed:  ,footer need  some change in css for example :smile:  
```
 position: absolute; 

```

**Review by Ghadeer93 — COMMENTED** [review]
css Footer

**Review by Ghadeer93 — CHANGES_REQUESTED** [review]
css footer

**Review by walaamedhat — DISMISSED** [review]

File: public/js/menFashion.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function btnClick() {
```
Comment (muhmushtaha) [inline public/js/menFashion.js:1]: When clicking on the second item the effects apply to the first item!

**Review by muhmushtaha — CHANGES_REQUESTED** [review]

File: public/js/menFashion.js (hunk 1 above)
Comment (Salam-Dalloul) [reply public/js/menFashion.js:1]: we solve it with the saved fashion feature :D

**Review by Ghadeer93 — APPROVED** [review]

**Pull request merged**